### PR TITLE
Modinreach-378 Orchid Patch Work

### PR DESCRIPTION
**Pull request by pavankumar181**
## Purpose
The Purpose of this PR is to Release patch for orchid release.

## Approach
Added All the changes whichever are needed for Orchid Patch Release.

#### TODOS and Open Questions

- [ ] Check logging

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### Stories
 * [MODINREACH-358](https://issues.folio.org/browse/MODINREACH-358) - No "Delivery stop" token value in INN-Reach paging slips
-* [MODINREACH-360](https://issues.folio.org/browse/MODINREACH-360) - Updated folio-spring-base.version to 6.1.0
+* [MODINREACH-360](https://iss.folio.org/browse/MODINREACH-360) - Updated folio-spring-base.version to 6.1.0
```
Comment (gurleenkaurbp) [inline NEWS.md]: Spelling of issues

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -16,19 +16,19 @@
     },
     {
       "id": "users",
-      "version": "15.3 16.0"
+      "version": "16.0"
```
Comment (gurleenkaurbp) [inline descriptors/ModuleDescriptor-template.json]: Use this -- "version": "15.3 16.0"

File: descriptors/ModuleDescriptor-template.json [hunk 3]
```diff
@@ -16,19 +16,19 @@
...
+      "version": "12.0"
     },
     {
       "id": "service-points-users",
       "version": "1.0"
     },
     {
       "id": "inventory-view",
-      "version": "2.0"
+      "version": "1.1"
```
Comment (gurleenkaurbp) [inline descriptors/ModuleDescriptor-template.json]: Use "version": "13.0"

File: descriptors/ModuleDescriptor-template.json [hunk 4]
```diff
@@ -28,7 +28,7 @@
     },
     {
       "id": "inventory-view",
-      "version": "1.1"
+      "version": "13.0"
```
Comment (gurleenkaurbp) [inline descriptors/ModuleDescriptor-template.json]: Here the version should be "version": "2.0"

Comment (gurleenkaurbp) [reply descriptors/ModuleDescriptor-template.json]: And the inventory version should be "version": "13.0"

Comment (pavankumar181) [reply descriptors/ModuleDescriptor-template.json]: changed

**Review by gurleenkaurbp — APPROVED** [review]

**Pull request merged**